WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Package Selection/Package Creation/ShippingLabelAddNewPackageViewModel.swift

```diff
@@ -0,0 +1,17 @@
+import UIKit
+import SwiftUI
+
+/// View model for `ShippingLabelAddNewPackage`.
+///
+final class ShippingLabelAddNewPackageViewModel: ObservableObject {
+    @Published var selectedIndex: Int
+
+    init(_ selectedIndex: Int = SelectedIndex.customPackage.rawValue) {
+        self.selectedIndex = selectedIndex
+    }
+
+    enum SelectedIndex: Int {
+        case customPackage = 0
+        case servicePackage = 1
```

Comment:
Quick tip: For enums with Int as raw value, the values starts from 0 by default and increase 1 by 1 for each of the next cases. But I appreciate the clarity here, so I think there's no need for removing the explicit values 👍